keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -514,310 +481,126 @@ spec:
       - "gpu"
 ```
 
-This request triggers resource allocation on a node that has a GPU device with
-2Gi of memory available and then the Pod runs on that node. The remaining
-capacity of the GPU may be usable for other pods, with constrains like alignment
-to segment sizes ensured by the resource driver.
-The lifecycle of the resource
-allocation is tied to the lifecycle of the Pod.
+This request triggers allocation on a node that has a GPU device with
+at least 2Gi of memory available and then the Pod runs on that node. The remaining
+capacity of the GPU may be usable for other pods. The lifecycle of the allocation
+is tied to the lifecycle of the Pod.
```

Comment:
Maybe I'm reading this wrong, but this is *not* how it will work with NVIDIA GPUs. You either get the whole GPU or you don't. Or do you mean that other pods could reference this _same_ claim in order to use the remaining memory (rather than instantiating a separate claim)?

Reply:
This is an abstract use case, not something that necessarily maps to an NVIDIA GPU.

I think it was originally written for classic DRA. We currently cannot do the "substract GPU RAM and use the rest" that is implied here.

I need to rewrite this based on static partitioning.